src/components/DataTable.js

```diff
@@ -20,7 +20,9 @@ class DataTable extends React.Component {
                   Header: "Item",
                   accessor: row => `${row.Item}`,
                   filterMethod: (filter, row) => {
-                    return row._original.Item.startsWith(filter.value);
+                    return row._original.Item.toLowerCase().includes(
+                      filter.value
```

Comment:
can you make the filter value also lowercase so that it still matches if they type with uppercase